docs/_themes/clusterhq/layout_docs.html

```diff
@@ -65,6 +65,8 @@
             })();
         </script>
     {% endif %}
+	
+    <link rel="shortcut icon" href="https://clusterhq.com/assets/favicon.ico"/>
```

Comment:
The favicon living in two different places doesn't really make sense in my opinion unless we were to have two different images. I guess it makes sense to put some policy on the website however to ensure this favicon doesn't move.

As for `//`, the website will **always** redirect to https so I guess it doesn't matter so much. Also if the docs are built locally and the client uses something like `file://`, the favicon would be broken (but that's maybe a case for adding the favicon locally?).

For the time I think this PR is fine to merge, if we choose that in fact the favicon should be the bird logo (something not the same as the website) or for some other reason it should be enclosed in this repo, it's a quick change that I am happy to handle.